cmd/opentelemetry/app/receiver/jaegerreceiver/jaeger_receiver.go

```diff
@@ -111,6 +112,29 @@ func configureCollector(v *viper.Viper, cfg *jaegerreceiver.Config) {
 			Endpoint: cOpts.CollectorHTTPHostPort,
 		}
 	}
+
+	// if at least one option is set, we assume they
```

Comment:
This sentence seems incomplete.

Reply:
That's what you get when you try to document your code and a kid enters the room...